.github/workflows/ci.yml

```diff
@@ -2,6 +2,8 @@ name: CI
 
 on:
   push:
+    tags:
+      - '*'
```

Comment:
Have you thought about using the [release event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release)?

We do it also in other projects.

Reply:
I was under the impression the issue was targeting creating an image on each tag, not only releases. But maybe I've misinterpreted it, so we could use [a trigger like this one](https://github.com/safe-global/safe-transaction-service/blob/cd330d0c0c81bdf2cbd233ec7564f4b5a03ce2c9/.github/workflows/python.yml#L8) and just generate images on release events, what is your opinion on this?

Reply:
> I was under the impression the issue was targeting creating an image on each tag, not only releases. But maybe I've misinterpreted it, so we could use [a trigger like this one](https://github.com/safe-global/safe-transaction-service/blob/cd330d0c0c81bdf2cbd233ec7564f4b5a03ce2c9/.github/workflows/python.yml#L8) and just generate images on release events, what is your opinion on this?

Something like that works. Regarding the issue, maybe it was not clear. Not that this doesn't work but we should compare different solutions. [The Safe Config Service also uses the release event](https://github.com/safe-global/safe-config-service/blob/3877fdc19a9e3109b0f7515025d9c3cecf099c58/.github/workflows/test.yml#L11-L12)

Reply:
I've took the Config Service as model to unify both services pipeline (also renaming `docker-publish` action to `docker-publish-staging`) in [494a482](https://github.com/safe-global/safe-client-gateway-nest/pull/525/commits/494a482d3dfb77fdcf3eed5ca9ab5a3e4b7ba808)